parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh

```diff
@@ -183,9 +168,10 @@ downloadContainerd() {
 {{- end}}
 
 installMoby() {
+    ensureRunc ${RUNC_VERSION:-""} # RUNC_VERSION is an optional override supplied via NodeBootstrappingConfig api
```

Comment:
this is the real change, runc needs to be installed first or else existing vhd version causes conflict with containerd.